client/my-sites/checkout/checkout/checkout-container.jsx

```diff
@@ -24,7 +22,6 @@ class CheckoutContainer extends React.Component {
 	state = {
 		headerText: '',
 		subHeaderText: '',
-		shouldShowGuaranteeSeal: false,
 		isJetpackNotAtomic: false,
```

Comment:
We don't need isJetpackNotAtomic prop too(added in #35404)

Reply:
Removed in https://github.com/Automattic/wp-calypso/pull/35728/commits/4d27869d6849e927e495261f64f6d0667da649b8